Themes/default/languages/LightPortal/.spanish_es.php

```diff
@@ -0,0 +1,263 @@
+$txt['lp_page_editor_type_default'] = 'El tipo de editor de la página por defecto';
+$txt['lp_hide_blocks_in_admin_section'] = 'Ocultar bloques activos en el área de administración';
+$txt['lp_open_graph'] = 'Open Graph';
+$txt['lp_page_og_image'] = 'Usa una imagen del contenido de la página';
+$txt['lp_page_og_image_set'] = array('ninguno', 'Primero encontrado', 'Último encontrado');
+$txt['lp_page_itemprop_address'] = 'Dirección de su organización';
+$txt['lp_page_itemprop_phone'] = 'Teléfono de su organización';
+
+// Plugins
+$txt['lp_plugins'] = 'Plugins';
+$txt['lp_plugins_desc'] = 'Puede activar o desactivar cualquiera de los plugins. ¡Y algunos de ellos también están personalizados!';
+$txt['lp_plugins_info'] = 'Los plugins instalados portal se enumeran aquí. Siempre puedes crear uno nuevo usando <a href="%1$s" target="_blank" rel="noopener">las instrucciones</a>.';
+
+$txt['lp_plugins_hooks_types'] = array(
+	'block' => 'Bloque',
+	'editor' => 'Editor',
+	'comment' => 'Widget de comentarios',
+	'parser' => 'Analizador de contenido',
+	'article' => 'Procesamiento de artículos',
+	'other' => 'Otro'
+);
+
+// Actions
+$txt['lp_title'] = 'Título';
+$txt['lp_actions'] = 'Acciones'; // Review later
+$txt['lp_action_on'] = 'Activar';
+$txt['lp_action_off'] = 'Desactivar';
+$txt['lp_action_clone'] = 'Clonar';
+$txt['lp_action_move'] = 'Mover';
+$txt['lp_read_more'] = 'Leer más...';
+
+// Blocks
+$txt['lp_blocks'] = 'Bloques';
+$txt['lp_blocks_manage'] = 'Administrar bloques';
+$txt['lp_blocks_manage_tab_description'] = 'Todos los bloques del portal creados se enumeran aquí. Para agregar un bloque, use el botón correspondiente.';
+$txt['lp_blocks_add'] = 'Agregar bloque';
+$txt['lp_blocks_add_title'] = 'Adding block'; // Review later
+$txt['lp_blocks_add_tab_description'] = 'Todavía no hay muchos bloques, pero existen los más universales ~ juega con ellos :)';
+$txt['lp_blocks_add_instruction'] = 'Seleccione el bloque deseado haciendo clic en él.';
+$txt['lp_blocks_edit_title'] = 'Edición de bloques';
+$txt['lp_blocks_edit_tab_description'] = $txt['lp_blocks_add_tab_description'];
+$txt['lp_block_content'] = 'Contenido';
+$txt['lp_block_icon_cheatsheet'] = 'Lista de los iconos';
+$txt['lp_block_type'] = 'Tipo de bloque';
+$txt['lp_block_priority'] = 'Prioridad';
+$txt['lp_block_icon_type'] = 'Tipo de icono';
+$txt['lp_block_icon_type_set'] = array('fas' => 'Sólido', 'lejos' => 'Regular', 'fabuloso' => 'Marcas'); // Review later
+$txt['lp_block_placement'] = 'Colocación';
+$txt['lp_block_placement_set'] = array(
+	'header' => 'Encabezado',
+	'top' => 'Centro (arriba)',
+	'left' => 'Lado izquierdo',
+	'right' => 'Lado derecho',
+	'bottom' => 'Centro (abajo)',
+	'footer' => 'Pie de página'
+);
+
+$txt['lp_block_areas'] = 'Acciones';
+$txt['lp_block_areas_subtext'] = 'Especifique una o más áreas (separadas por comas) para mostrar el bloque en:<br>
+<ul>
```

Comment:
Add class="bbc_list" for **ul**.

Reply:
Apply the change in Latin but not in es, it is already fixed.